nixos/modules/services/audio/spotifyd.nix

```
@@ -4,7 +4,15 @@ with lib;
 
 let
   cfg = config.services.spotifyd;
-  spotifydConf = pkgs.writeText "spotifyd.conf" cfg.config;
+  toml = pkgs.formats.toml {};
+  warnConfig =
+    if cfg.config != ""
+    then lib.trace "Using the stringly typed .config attribute is discouraged. Use the TOML typed .settings attribute instead."
+    else identity;
```

I get

```
error: undefined variable 'identity' at /home/anders/nixpkgs/nixos/modules/services/audio/spotifyd.nix:11:10
```

Did you mean the below? Everything seems to work correctly for me with the below change.

```suggestion
    else id;
```

---

Yes, of course, sorry.